src/components/project/project.service.ts

```diff
@@ -293,19 +293,12 @@ export class ProjectService {
       'project'
     );
 
-    if (changes.step && stepValidation) {
-      await this.projectRules.verifyStepChange(
-        input.id,
-        session,
-        changes.step,
-        changeset
-      );
-    }
-
     const {
       primaryLocationId,
       marketingLocationId,
       fieldRegionId,
+      step, // skip step, use this.updateStep
+      status, // skip status, use this.updateStep
```

Comment:
change step via updateProject is deprecated but we still need to support it for a little bit.